src/Spice86.Core/Emulator/OperatingSystem/LocalFileSearchManager.cs

```diff
@@ -0,0 +1,134 @@
+﻿namespace Spice86.Core.Emulator.OperatingSystem;
+
+using System.Runtime.CompilerServices;
+
+public class LocalFileSearchManager {
```

Comment:
This class should be merged with the DosPathResolver.

In the future, the DosPathResolver will be abstracted away (think of IOCTL when MSCDEX or floppy images are involved), or at least its direct usage of the System.IO namespace.